### ISSUE TITLE
app crash on launch

### ISSUE THREAD

**Issue by SpringHgui** [flutter/flutter#105504]
app crash on launch
```
06-07 15:13:13.728  1471  1471 D Launcher2.2.10: flow not clicked com.example.flutter_application_1com.example.flutter_application_1.MainActivity
06-07 15:13:13.728  1471  1471 D Launcher2.2.10: flow click desktop com.example.flutter_application_1com.example.flutter_application_1.MainActivity
06-07 15:13:13.729   421   793 I ActivityManager: START u0 {act=android.intent.action.MAIN flg=0x10200000 cmp=com.example.flutter_application_1/.MainActivity} from uid 10072
06-07 15:13:13.737   421   793 E ActivityManager: getPackageFerformanceMode--ComponentInfo{com.example.flutter_application_1/com.example.flutter_application_1.MainActivity}----com.example.flutter_application_1
06-07 15:13:13.745   421  1383 E ActivityManager: getPackageFerformanceMode--ComponentInfo{com.example.flutter_application_1/com.example.flutter_application_1.MainActivity}----com.example.flutter_application_1
06-07 15:13:13.763   421  1383 I ActivityManager: Start proc 11535:com.example.flutter_application_1/u0a94 for activity com.example.flutter_application_1/.MainActivity
06-07 15:13:13.992 11535 11535 E AndroidRuntime: Process: com.example.flutter_application_1, PID: 11535
06-07 15:13:13.992 11535 11535 E AndroidRuntime: java.lang.RuntimeException: Unable to instantiate activity ComponentInfo{com.example.flutter_application_1/com.example.flutter_application_1.MainActivity}: java.lang.ClassNotFoundException: Didn't find class "com.example.flutter_application_1.MainActivity" on path: DexPathList[[zip file "/data/app/com.example.flutter_application_1-KTLaXS2BI_DGEpBaDH4xDg==/base.apk"],nativeLibraryDirectories=[/data/app/com.example.flutter_application_1-KTLaXS2BI_DGEpBaDH4xDg==/lib/arm, /data/app/com.example.flutter_application_1-KTLaXS2BI_DGEpBaDH4xDg==/base.apk!/lib/armeabi-v7a, /system/lib, /vendor/lib, /system/lib, /vendor/lib, /epay/usr/lib, /epay/lib, /system/vendor/lib, /home/EPT_Middleware/lib]]
06-07 15:13:13.992 11535 11535 E AndroidRuntime: Caused by: java.lang.ClassNotFoundException: Didn't find class "com.example.flutter_application_1.MainActivity" on path: DexPathList[[zip file "/data/app/com.example.flutter_application_1-KTLaXS2BI_DGEpBaDH4xDg==/base.apk"],nativeLibraryDirectories=[/data/app/com.example.flutter_application_1-KTLaXS2BI_DGEpBaDH4xDg==/lib/arm, /data/app/com.example.flutter_application_1-KTLaXS2BI_DGEpBaDH4xDg==/base.apk!/lib/armeabi-v7a, /system/lib, /vendor/lib, /system/lib, /vendor/lib, /epay/usr/lib, /epay/lib, /system/vendor/lib, /home/EPT_Middleware/lib]]
06-07 15:13:13.999   421   793 W ActivityManager:   Force finishing activity com.example.flutter_application_1/.MainActivity
06-07 15:13:14.009   421   442 I ActivityManager: Showing crash dialog for package com.example.flutter_application_1 u0
06-07 15:13:14.185   633  4670 W asset   : Asset path /data/app/com.example.flutter_application_1-3Z2aO86qxF5cCp-Dv8b18Q==/base.apk is neither a directory nor file (type=1).
06-07 15:13:14.185   633  4670 E ResourcesManager: failed to add asset path /data/app/com.example.flutter_application_1-3Z2aO86qxF5cCp-Dv8b18Q==/base.apk
06-07 15:13:14.186   633  4670 W PackageManager: Failure retrieving resources for com.example.flutter_application_1
06-07 15:13:14.506   421   441 W ActivityManager: Activity pause timeout for ActivityRecord{1992b7e u0 com.example.flutter_application_1/.MainActivity t47 f}
06-07 15:13:25.512   421   441 W ActivityManager: Activity destroy timeout for ActivityRecord{1992b7e u0 com.example.flutter_application_1/.MainActivity t47 f}
```

**Comment by exaby73** [comment]
Hi @SpringHgui, please provide a [minimal, reproducible example][1] and the output of `flutter doctor -v`.

[1]: https://stackoverflow.com/help/minimal-reproducible-example

**Comment by Abdullah2100** [comment]
please provide more information

**Comment by exaby73** [comment]
Hello @SpringHgui. I noticed you closed the issue. Have you found a suitable solution to your issue? If so, would you kindly post your solution so that anyone else with the same error as you would be able to solve it as well.

**Comment by SpringHgui** [comment]
The first project I created using the vscode plugin according to the tutorial, in debug mode, the above problem occurs, and when this problem occurs, it will also cause my PC keyboard to fail globally, I have to restart the PC to operate the keyboard, the same way to create The rest of the projects did not reproduce the problem. Finally, I executed clean in the first project and started the project with F5. It was all good. I don't know why.

run clean，solved
```
flutter clean
```
